tests/unit/test_testutils.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import mock
+
+from awscli.testutils import create_bucket
+from awscli.testutils import BaseCLIDriverTest
+
+
+class TestCreateBucket(BaseCLIDriverTest):
```

Comment:
Personally I don't do it in a "black or white" way. I sometimes wrote tests for test utility when I feel a need, and sometimes - oh God of Test pls forgive me - skip test cases for simple function.

The funny story in this case is that, this test actually helped me, twice. At first it helped me to realize that, by default the exception is `awscli.errorhandler.ClientError` rather than `botocore.exceptions.ClientError`. And then it helped me just now to use `client.meta.events.unregister(...)` rather than `session.unregister(...)`. Those two errors won't be discovered by syntax check. Ouch. Ideally it shouldn't need a test case to get this seemingly-simple task right, but that is a different story anyway.